### PR TITLE
[Component] Card

### PR DESCRIPTION
**Pull request by foliveira**
Pulled from #8

### CODE REVIEW

File: packages/card/card.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Text, View, StyleSheet, Dimensions } from "react-native";
+import ArticleSummary from "@times-components/article-summary";
```
Comment (craigbilner) [inline packages/card/card.js]: timescomponents

File: packages/card/card.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Text, View, StyleSheet, Dimensions } from "react-native";
```
Comment (craigbilner) [inline packages/card/card.js]: unused imports

File: packages/card/card.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import "react-native";
+import React from "react";
+import Card from "./card";
+import renderer from "react-test-renderer";
+
+it("renders correctly", () => {
+  const tree = renderer.create(<Card />).toJSON();
+
+  expect(tree).toBeTruthy();
```
Comment (craigbilner) [inline packages/card/card.test.js]: Can we do snapshot testing?

File: packages/card/card.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      paddingRight: isHorizontal ? 17 : 0
+    },
+    summaryContainer: {
+      paddingRight: isHorizontal ? "8%" : 0,
+      width: isHorizontal ? "60%" : "100%"
+    }
+  });
+
+class CardComponent extends React.Component {
+  constructor(...args) {
```
Comment (sericaia) [inline packages/card/card.js]: replace ...args with props? 👍

**Pull request merged**